sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/aad/configuration/properties/AadAuthenticationProperties.java

```diff
@@ -111,41 +111,41 @@ public class AadAuthenticationProperties implements InitializingBean {
     private int jwtSizeLimit = RemoteJWKSet.DEFAULT_HTTP_SIZE_LIMIT; /* bytes */
 
     /**
-     * The lifespan(duration) of the cached JWK set before it expires. The default value is `5m`.
+     * The lifespan(duration) of the cached JWK set before it expires.
```

Comment:
Could you help check how the `The default value is `5m`` is auto-generated by the doc plugin?